### PR TITLE
Astf cleanup

### PR DESCRIPTION
**Pull request by Emil-Gustafsson**

### CODE REVIEW

**Comment by Emil-Gustafsson** [comment]
should be no functionality change, only removed unused stuff, but best if it could be tested first

**Comment by Emil-Gustafsson** [comment]
ok to merge?

File: src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFCapInfo.java [hunk 1]
```diff
@@ -43,12 +43,9 @@ private void paramCheck() {
...
             String.format("bad param combination,l7Percent %s ,cps %s ", l7Percent, cps));
       }
-      l7Percent = l7Percent;
-      cps = cps;
+
     } else {
-      if (cps > 0) {
-        cps = cps;
-      } else {
+      if (cps < 0) {
```
Comment (leomaatEric) [inline src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFCapInfo.java]: if (cps <= 0)

File: src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFProfile.java [hunk 2]
```diff
@@ -17,15 +17,11 @@
...
-  private ASTFIpGen astfIpGen;
   private ASTFGlobalInfo astfClientGlobalInfo;
   private ASTFGlobalInfo astfServerGlobalInfo;
   private List<ASTFTemplate> astfTemplateList;
-  private List<ASTFCapInfo> astfCapInfoList;
-
-  private Map<String, Integer> tgName2TgId =
-      new LinkedHashMap<>(); // template group name -> template group id
+  private Map<String, Integer> tgName2TgId = new LinkedHashMap<>(); // template group name ->
+  // template group id
```
Comment (leomaatEric) [inline src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFProfile.java]: improper format ,  no need of line break

Comment (Emil-Gustafsson) [reply src/main/java/com/cisco/trex/stateful/api/lowlevel/ASTFProfile.java]: when i edit the code it is already one line

**Pull request merged**